django-verdant/rca/static/rca/css/streamfield-blocks/events.less

```diff
@@ -0,0 +1,203 @@
+            margin: 0 116px @spacing-one 0;
+        }
+
+        &:hover {
+            color: @color-pink;
+        }
+    }
+
+    // v similar to news arrow - refactor if time
+    &__more-arrow {
+        width: 39px;
+        height: 30px;
+        margin-left: @spacing-one;
+        display: inline-block;
+        fill: @color-white;
+
+        .events-block__more:hover & {
+            fill: @color-pink;
+        }
+    }
+
+    // carousel arrows
+
+    &__button {
+        //reset global button styles
+        line-height: 0;
+        padding: 0;
+        display: block;
+        text-transform: none;
+        background-color: transparent;
+        border: 0;
+        font-size: 0;
+
+        &:hover {
+            background-color: transparent;
+        }
+
+        // apply button styles
+        position: absolute;
+        background-image: none;
+        width: 27px;
+        height: 49px;
+        z-index: 1;
+
+        @media @media-desktop-regular {
+            width: 54px;
+            height: 98px;
+        }
+    }
+
+    &__arrow {
+        width: 27px;
+        height: 49px;
+        fill: @color-white;
+
+        @media @media-desktop-regular {
+            width: 54px;
+            height: 98px;
+        }
+
+        &:hover {
+            fill: @color-pink;
+        }
+
+        .events-block__prev & {
+            transform: scaleX(-1);
+        }
+    }
+
+    &__next {
+        right: 4px;
+        top: 30%;
+
+        @media @media-desktop-regular {
+            right: 22px;
+            top: 10%;
+        }
+
+        @media @media-desktop-large {
+            top: 15%;
+        }
+    }
+
+    &__prev {
+        left: 4px;
+        top: 30%;
+
+        @media @media-desktop-regular {
+            left: 22px;
+            top: 10%;
+        }
+
+        @media @media-desktop-large {
+            top: 15%;
+        }
+    }
+
+    // slick overrides
+
+    // the 2 inactive slides (partially hidden) - black and inactive
+    .slick-slide:not(.slick-active) {
+        pointer-events: none;
+
+        .event-item__image-wrapper {
+            position: relative;
+
+            &:after {
+                content: '';
+                width: 100%;
+                height: 100%;
+                position: absolute;
+                top: 0;
+                background-color: @color-black;
+
+                @media @media-desktop-regular {
+                    .opacity(60);
+                }
+            }
+        }
+
+        .event-item__dates {
+            display: none;
+        }
+    }
+}
+
+.event-item {
+    display: block;
+    padding: 0 3px;
+
+    @media @media-desktop-regular {
+        padding: 0 @spacing-half;
+    }
+
+    &__wrapper {
+        position: relative;
+        margin-bottom: @spacing-one;
+    }
+
+    &__dates {
+        position: absolute;
+        top: 0;
+        left: 0;
+        background-color: #A4429C; // design doesn't match existing colour palette
+        .opacity(90);
```

Comment:
I noticed you were using this before, is there a reason to not just use `opacity: 0.9;`?

Reply:
The mixin adds the filter fallback, as when the site was first built we were including support for older browsers. I don't think there's any harm in continuing to use it for consistency.